.circleci/config.yml

```diff
@@ -67,32 +67,32 @@ commands:
             # The release notification only makes sense on tagged commits. If the commit is untagged, just bail out.
             [[ "<< parameters.event >>" == "release" ]] && { [[ $CIRCLE_TAG != "" ]] || { echo "Not a tagged commit - notification skipped."; exit 0; } }
 
-            curl "https://api.gitter.im/v1/rooms/${GITTER_NOTIFY_ROOM_ID}/chatMessages" \
+            curl "https://${MATRIX_SERVER}/_matrix/client/v3/rooms/${MATRIX_NOTIFY_ROOM_ID}/send/m.room.message" \
               --request POST \
               --include \
               --header "Content-Type: application/json" \
               --header "Accept: application/json" \
-              --header "Authorization: Bearer ${GITTER_API_TOKEN}" \
-              --data "{\"text\":\"${message}\"}"
+              --header "Authorization: Bearer ${MATRIX_ACCESS_TOKEN}" \
+              --data "{\"msgtype\":\"m.text\", \"body\":\"${message}\"}"
```

Comment:
Nice that this is still simple. I was afraid we'd have to jump over more hoops :)